Dockerfile.db_migrations

```diff
@@ -1,11 +1,11 @@
-ARG TAG=
+ARG TAG=9.10
```

Comment:
Is there a reason we default to `9.10` here but `9.10-alpine` in `Dockerfile.db_clean`? Perhaps this was added when building locally?

Reply:
TIL: Defining the arg in the dockerfile still allows it to be overriden, this is basically just the default value. 

@Jdwoodson Can you:
- Add `9.10` to the DB Clean Dockerfile
- Modify the pipeline to build `9.10-alpine` for deploying to ECS
- Remove references to `9.10` in the compose files (since we shouldn't need to specify it if it's in the dockerfile, only if we want to build a different image, i.e. for ECS)

Reply:
@ClayBenson94 correct about the default ARG value. Just makes the dockerfile complete to run.

There was a reason I went with alpine for clean and not for migrations. trying to find my notes on it because I thought I marked down why.

Reply:
TL;DR for historical purposes - db_clean runs only in CI and we can use alpine image successfully 100% of time. db_migrations runs locally and in ci.  As part of local dev process, using mutli-arch image allows for devs on different architectures to not have to specifically define their arch. in CI, we can use alpine 100%.